modules/MultiDRM/MultiDRM.loader.js

```diff
@@ -140,7 +140,7 @@
 			var hlsSource = sources.filter( function ( source ) {
 				return ( source.mimeType === "application/vnd.apple.mpegurl" );
 			});
-			drmSources.push(hlsSource);
+			drmSources.concat(hlsSource);
 		}
```

Comment:
@OrenMe - so hlsSource might contain more than one source? In that case, maybe call it hlsSources. but feel free to merge if you don't feel like changing the variable name...